packages/neos-ui-editors/src/Editors/TextField/index.js

```diff
@@ -41,7 +41,7 @@ export default class TextField extends PureComponent {
             className={className}
             id={id}
             autoFocus={finalOptions.autoFocus}
-            value={value}
+            value={value === null ? '' : value}
```

Comment:
i must say i dont fully understand this line 😂

Reply:
btw one of the "implications" being that even if the value changed from "foo" to null, it would not update the textarea